lib/request_cache/middleware.ex

```diff
@@ -6,9 +6,20 @@ if RequestCache.Application.dependency_found?(:absinthe) and RequestCache.Applic
 
     @impl Absinthe.Middleware
     def call(%Absinthe.Resolution{} = resolution, opts) when is_list(opts) do
+      opts = ensure_valid_ttl(opts)
       enable_cache_for_resolution(resolution, opts)
     end
 
+    @impl Absinthe.Middleware
+    def call(%Absinthe.Resolution{} = resolution, ttl) when is_binary(ttl) do
+      enable_cache_for_resolution(resolution, ttl: RequestCache.Config.default_ttl())
+    end
+
+    @impl Absinthe.Middleware
+    def call(%Absinthe.Resolution{} = resolution, ttl) when is_nil(ttl) do
+      enable_cache_for_resolution(resolution, ttl: RequestCache.Config.default_ttl())
+    end
+
```

Comment:
These probably aren't necessary given that these would throw a FunctionClauseError which is probably more desirable, particularly for the is_binary case.

Reply:
Wait why is TTL binary? Shouldn't it be is_integer?

Reply:
I was covering all the bases, but agree with Nick